docs/sphinx/users/comlinetools/conversion.txt

```diff
@@ -67,3 +67,16 @@ example, to use LZW compression in a TIFF file:
 
   bfconvert -compression LZW /path/to/input output-lzw.tiff
 
+If the specified output file already exists, bfconvert will prompt to
```

Comment:
I don't know if we want `:command:` output for these, maybe not; just a thought.